src/LedControllerFactory.cpp

```diff
@@ -64,7 +64,7 @@ uint8_t LedControllerFactory::determineLedType(bool lightBar) {
     } else {
         ledFreqStr = std::string(AppConfiguration::getInstance()->config.ledFrequency.c_str());
     }
-    if (ledFreqStr == "KHZ800") {
+    if (ledFreqStr == "800kHz") {
```

Comment:
Please change it to support both "KHZ800" and "800kHz" to not break with older version of the App